lib/config/account/index.js

```diff
@@ -0,0 +1,22 @@
+module.exports = accountConfig
+
+var defaultsDeep = require('lodash').defaultsDeep
+
+function accountConfig (state, callback) {
+  var usersDb = new state.config.PouchDB(state.db.authenticationDb)
+  usersDb.installUsersBehavior()
```

Comment:
Have been thinking about this a bit more...

What happens if the plugin is already registered? Giving the user power to define their own PouchDB constructor also means they're likely using their own plugins, and wouldn't be surprised if they need this plugin somewhere down the line.

Binding it here may mean that users have to wait until `@hoodie/server` has been registered to take advantage of the `pouchdb-users` plugin. So I suppose either way, we need to document that it's included (therefore don't try to register it), or that it's not - and therefore the user needs to register it.